MapManager/MapManager_Reverse.cpp

```diff
@@ -1,6 +1,41 @@
 #include "MapManager.h"
 
-//���Ӳ���ɵ�ͼ�ķ�ת
+bool dir[8][2]={{0,1},{0,-1},{1,0},{-1,0},{1,1},{1,-1},{-1,1},{-1,-1}};
```

Comment:
bool?
maybe it's better to use int here.

Reply:
confirm.